usr.sbin/bsdinstall/scripts/jail

```diff
@@ -137,6 +137,7 @@ trap error SIGINT	# SIGINT is bad again
 bsdinstall config  || error "Failed to save config"
 cp /etc/resolv.conf $1/etc
 cp /etc/localtime $1/etc
+cp /var/db/zoneinfo $1/var/db
```

Comment:
Two coments: first, these are two different programs, so should have two separate commits
Second, in the sysbuild patch you checked to see if this exists, but here you do not, why?

Reply:
I was just following the previous "style".

I would prefer to check if the file exists first as well. However, since this is `bsdinstall(8)` one might think the file is not there or you want to overwrite anyway.

Reply:
I was thinking more basic: what if the host doesn't have it... but now that I'm thinking about it, there's bigger issues if the host doesn't have it...